### PR TITLE
QS 2022 for practice 

### PR DESCRIPTION
**Pull request by shuqin32**
- automated  droppable box

### CODE REVIEW

**Review by qingyuanre1 — APPROVED** [review]
good job!

**Review by ajf876 — APPROVED** [review]
Looks good.

File: src/test/java/selenium/Droppable.java [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+		}
+	}
+
+	public static void customeWait(int second) {
+		try {
+			Thread.sleep(second * 1000);
+		} catch (InterruptedException e) {
+			e.printStackTrace();
+		}
+	}
```
Comment (Muradil-Erkin) [inline src/test/java/selenium/Droppable.java:47]: In our Utilities class, we already have this function, can you use that one please?

**Review by Muradil-Erkin — CHANGES_REQUESTED** [review]

**Review by Muradil-Erkin — APPROVED** [review]
LGTM!

**Pull request merged**